sdk/go/pulumi/config/try.go

```diff
@@ -27,7 +27,8 @@ import (
 func Try(ctx *pulumi.Context, key string) (string, error) {
 	v, ok := ctx.GetConfig(key)
 	if !ok {
-		return "", fmt.Errorf("missing required configuration variable '%s'; run `pulumi config` to set", key)
+		return "",
```

Comment:
cc @EvanBoyle

Reply:
If you run `make gen` in the go SDK you can see that these files are actually out of sync on master. We should check this in.

Surprised that a dirty git branch doesn't fail any of our builds.

Reply:
I was also somewhat surprised that the dirty checking did not pick this up, but it is in fact a correct change so far as I can tell.